File: java-play-framework/app/apimodels/NonExistingStreamError.java
package apimodels;

import apimodels.CodedErrorBase;
import apimodels.NonExistingStreamErrorAllOf;
import com.fasterxml.jackson.annotation.*;
import java.util.Set;
import javax.validation.*;
import java.util.Objects;
import javax.validation.constraints.*;
/**
 * NonExistingStreamError
 */
@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaPlayFrameworkCodegen", date = "2021-08-08T20:41:53.018098Z[Etc/UTC]")
@SuppressWarnings({"UnusedReturnValue", "WeakerAccess"})
public class NonExistingStreamError   {
  @JsonProperty("result")
  @NotNull

  private Object result = null;

  @JsonProperty("msg")
  @NotNull

  private Object msg = null;

  @JsonProperty("code")
  
  private Object code = null;

  @JsonProperty("stream")
  
  private String stream;

  public NonExistingStreamError result(Object result) {
    this.result = result;
    return this;
  }

   /**
   * Get result
   * @return result
  **/
  public Object getResult() {
    return result;
  }

  public void setResult(Object result) {
    this.result = result;
  }

  public NonExistingStreamError msg(Object msg) {
    this.msg = msg;
    return this;
  }

   /**
   * Get msg
   * @return msg
  **/
  public Object getMsg() {
    return msg;
  }

  public void setMsg(Object msg) {
    this.msg = msg;
  }

  public NonExistingStreamError code(Object code) {
    this.code = code;
    return this;
  }

   /**
   * Get code
   * @return code
  **/
  public Object getCode() {
    return code;
  }

  public void setCode(Object code) {
    this.code = code;
  }

  public NonExistingStreamError stream(String stream) {
    this.stream = stream;
    return this;
  }

   /**
   * The name of the stream that could not be found. 
   * @return stream
  **/
  public String getStream() {
    return stream;
  }

  public void setStream(String stream) {
    this.stream = stream;
  }


  @Override
  public boolean equals(Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    NonExistingStreamError nonExistingStreamError = (NonExistingStreamError) o;
    return Objects.equals(result, nonExistingStreamError.result) &&
        Objects.equals(msg, nonExistingStreamError.msg) &&
        Objects.equals(code, nonExistingStreamError.code) &&
        Objects.equals(stream, nonExistingStreamError.stream);
  }

  @Override
  public int hashCode() {
    return Objects.hash(result, msg, code, stream);
  }

  @SuppressWarnings("StringBufferReplaceableByString")
  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class NonExistingStreamError {\n");
    
    sb.append("    result: ").append(toIndentedString(result)).append("\n");
    sb.append("    msg: ").append(toIndentedString(msg)).append("\n");
    sb.append("    code: ").append(toIndentedString(code)).append("\n");
    sb.append("    stream: ").append(toIndentedString(stream)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: cpp-tiny/lib/Models/NonExistingStreamError.cpp


#include "NonExistingStreamError.h"

using namespace Tiny;

NonExistingStreamError::NonExistingStreamError()
{
	result = null;
	msg = null;
	code = null;
	stream = std::string();
}

NonExistingStreamError::NonExistingStreamError(std::string jsonString)
{
	this->fromJson(jsonString);
}

NonExistingStreamError::~NonExistingStreamError()
{

}

void
NonExistingStreamError::fromJson(std::string jsonObj)
{   
    bourne::json object = bourne::json::parse(jsonObj);

    const char *resultKey = "result";
    
    if(object.has_key(resultKey)) 
    {
        bourne::json value = object[resultKey];


        

        AnyType* obj = &result;
		obj->fromJson(value.dump());

    }

    const char *msgKey = "msg";
    
    if(object.has_key(msgKey)) 
    {
        bourne::json value = object[msgKey];


        

        AnyType* obj = &msg;
		obj->fromJson(value.dump());

    }

    const char *codeKey = "code";
    
    if(object.has_key(codeKey)) 
    {
        bourne::json value = object[codeKey];


        

        AnyType* obj = &code;
		obj->fromJson(value.dump());

    }

    const char *streamKey = "stream";
    
    if(object.has_key(streamKey)) 
    {
        bourne::json value = object[streamKey];


        
        jsonToValue(&stream, value, "std::string");


    }


}

bourne::json
NonExistingStreamError::toJson()
{
    bourne::json object = bourne::json::object();

    




	object["result"] = getResult().toJson();

    




	object["msg"] = getMsg().toJson();

    




	object["code"] = getCode().toJson();

    



    object["stream"] = getStream();



    return object;

}

AnyType
NonExistingStreamError::getResult()
{
	return result;
}

void
NonExistingStreamError::setResult(AnyType  result)
{
	this->result = result;
}

AnyType
NonExistingStreamError::getMsg()
{
	return msg;
}

void
NonExistingStreamError::setMsg(AnyType  msg)
{
	this->msg = msg;
}

AnyType
NonExistingStreamError::getCode()
{
	return code;
}

void
NonExistingStreamError::setCode(AnyType  code)
{
	this->code = code;
}

std::string
NonExistingStreamError::getStream()
{
	return stream;
}

void
NonExistingStreamError::setStream(std::string  stream)
{
	this->stream = stream;
}



